labgrid/driver/sshdriver.py

```diff
@@ -148,6 +154,9 @@ def _start_own_master_once(self, timeout):
                 format(subprocess_timeout, args),
             )
 
+        if self.networkservice.password and os.path.exists(pass_file):
+            os.remove(pass_file)
```

Comment:
This should be in a try:/finally: block.

Reply:
I tried to clean up earlier, directly after Popen, but then ssh complains about the missing file. I'll do a finally after the big try.